ClientApp/src/app/components/inplaces/date-inplace/date-inplace.component.ts

```diff
@@ -84,6 +84,7 @@ export class DateInplaceComponent implements OnInit {
     }
 
     //Check if date range does not make sense
+    
```

Comment:
seems like this newline is pointless, but I don't think it's a big enough deal to block approval

Reply:
Good catch, got rid of this